modules/mobile/db/migrate/20210913194827_create_mobile_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMobileUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :mobile_users do |t|
+      t.string :user_id, null: false
```

Comment:
For the initial use case of SM notification filtering, ICN will be the id. Which I think this is a holder for :) but it could be good to make it explicitly that, as there will be other user ids in the future.

Reply:
yeah, I used user_id since I didn't know what the other customers would be using and I was thinking we would still want them to be the same field from our perspective. Do you think them being the same field could cause issues in the future?